calicoctl/resourcemgr/bgpconfig.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourcemgr
+
+import (
+	"context"
+
+	api "github.com/projectcalico/libcalico-go/lib/apiv2"
+	client "github.com/projectcalico/libcalico-go/lib/clientv2"
+	"github.com/projectcalico/libcalico-go/lib/options"
+)
+
+func init() {
+	registerResource(
+		api.NewBGPConfiguration(),
+		api.NewBGPConfigurationList(),
+		false,
+		[]string{"bgpconfiguration", "bgpconfigurations", "bgpconfig", "bgpconfigs"},
+		[]string{"NAME"},
```

Comment:
include all of the fields for this one

Reply:
you might need to do some magic to handle nil values in the go template below. 

for now, maybe just display nil as a dash...?

Reply:
I have displayed `<nil>` as `----`.
Ex:
```
NAME         LOGSEVERITY   MESHENABLED   DEFAULTASN
default      Info          false         6511
testConfig   Info          ----          ----
```